### ISSUE TITLE
Q: force keep single item

### ISSUE THREAD

**Issue by DJafari** [mikepenz/FastAdapter#641]
hi @mikepenz 
how can possible keep single item with certain viewType from recycle ?

i want all items with `viewType: R.id.custom_view_type` always keep in list and don't destroy view

i google it, found two answer :

```java
recyclerView.getRecycledViewPool().setMaxRecycledViews(R.id.custom_view_type, 0);
```
don't work and unbindView of Item is called

second :

```java
holder.setIsRecyclable(false);
```

when i set this in `bindView` of item, `unbindView` not called, but when i reach to this item, `bindView` called again with adapter

in other words because my item bind is too expensive, i don't want rebind this item more than one

do you have any idea ?

thanks

**Comment by mikepenz** [comment]
What's the `expensive` part about this item? Perhaps you can cache the expensive part, and keep it flexible? 

(Sorry can't deep jump in right now)

**Comment by DJafari** [comment]
i have a long textview (Html.fromHtml) with many http image, sure images cached, but rebind of this caused a lag

**Comment by DJafari** [comment]
@mikepenz ok i found a way

```java
private ViewHolder viewHolder;

@Override
@NonNull
public ViewHolder getViewHolder(@NonNull View v) {
    if(viewHolder == null) {
        viewHolder = new ViewHolder(v);
        viewHolder.setIsRecyclable(false);
    }
    return viewHolder;
}
```

In your opinion it's ok ? not caused another problem ?

**Comment by mikepenz** [comment]
@DJafari should be fine. As the FastAdapter is not handling recycling. It is only providing items, and creating them. 

The RecyclerView decides if something is recycled, reused, ...

**Comment by DJafari** [comment]
@mikepenz good, thanks for your help

**Comment by mikepenz** [comment]
One thing. possibly setting the `setIsRecycleable(false)` in the `ctor` of this ViewHolder from this item is a more secure way.

Oh I just see.Is this item multiple times in the same RecyclerView?

**Comment by DJafari** [comment]
@mikepenz can i know what is difference?

no i use one item with this viewType

**Comment by mikepenz** [comment]
it just will ensure that this is always set :D and it is not forgotten somewhere. 

ok then it is ok. Remember. This item can not and should never be used in any place where it will exist multiple times in on activity, as it will always end up being the same viewholder
